src/pretalx/static/vendored/fullcalendar/locales/af.global.min.js

```diff
@@ -0,0 +1,6 @@
+/*!
+FullCalendar Core v6.1.5
+Docs & License: https://fullcalendar.io
```

Comment:
Quick check: please confirm that these are the files from the MIT licensed (not premium) version of fullcalendar!?

Reply:
The files are extracted from the GH release: https://github.com/fullcalendar/fullcalendar/releases/tag/v6.1.5
They are MIT license.